go-controller/pkg/ovn/service.go

```diff
@@ -0,0 +1,514 @@
+package ovn
```

Comment:
this commit is accidentally adding back the legacy service.go file. It shouldn't be here. I think you need to fix this revert commit.

Reply:
^